schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/layout/layout-custom-fields.xsl

```diff
@@ -128,6 +128,24 @@
   </xsl:template>
 
 
+  <xsl:template mode="mode-iso19115-3.2018"
```

Comment:
Is this related to the changes?

Reply:
Yes to display `@uuidref` as readonly fields